tests/foreman/api/test_contentmanagement.py

```diff
@@ -311,37 +307,43 @@ def test_positive_checksum_sync(self):
 
         :CaseImportance: Critical
         """
-        repomd_path = 'repodata/repomd.xml'
+        REPOMD_PATH = 'repodata/repomd.xml'
         # Create organization, product, lce and repository with sha256 checksum
         # type
-        org = entities.Organization(smart_proxy=[self.capsule_id]).create()
+        org = entities.Organization(smart_proxy=[capsule_vm._capsule.id]).create()
         product = entities.Product(organization=org).create()
         repo = entities.Repository(
             product=product, checksum_type='sha256', download_policy='immediate'
         ).create()
         lce = entities.LifecycleEnvironment(organization=org).create()
         # Associate the lifecycle environment with the capsule
-        capsule = entities.Capsule(id=self.capsule_id).read()
+        capsule = entities.Capsule(id=capsule_vm._capsule.id).read()
```

Comment:
`entities.Capsule(id=capsule_vm._capsule.id` should be equivalent to `capsule_vm._capsule`, I don't think there's an actual need to re-create the entity instance here.  But you're just continuing the pattern, changing this is optional.